contrib/pyln-client/requirements.txt

```diff
@@ -1 +1,2 @@
 recommonmark>=0.7.*
+pyln-bolt7
```

Comment:
It is better to fix the version here with >= latest one? Maybe pip can take the decision to use the old one if in the system there is an older version?

So we are putting here some dependencies that can cause errors in production,  I will preferer to have a `pyln-bolt7>=1.0.2.186`

Reply:
Hm, theres something about the pyln-spec dependency.

I put the `gossip_store_channel_amount` into the bolt7 spec package, not sure if I was supposed to, see  32cb69ef. but the one that will be installed with pip is still the old one which results in:

```
ImportError: cannot import name 'gossip_store_channel_amount' from 'pyln.spec.bolt7' 
```

So we need to put `bolt7 >= 1.0.2.187` (last digit incremented) since the 186 does not have the `gossip_store_channel_amount` in its csv

Reply:
187 is not released yet, but it will be the one with the changes we made in this branch... not sure how we handle something like this

Reply:
I did the pypi upload, even though we hadn't merged, because I needed it to fix lnprototest...

Reply:
Oh, I missed that change, it's completely wrong!

I will fix that a different way tomorrow...